### PR TITLE
feat(frontend): update isActive login of Indicator component

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

We need to update `isActive` logic of the Indicator component to smoothly animate on last-to-first and first-to-last transitions.

### CODE REVIEW

File: src/frontend/src/lib/components/carousel/Indicator.svelte [hunk 1]
```diff
@@ -4,9 +4,15 @@
 
 	export let currentSlide: number;
 	export let index: number;
+	export let totalSlides: number;
 
 	let isActive: boolean;
-	$: isActive = currentSlide === index;
+	$: isActive =
+		currentSlide === index ||
+		// set the last indicator as active on last-to-first transition
```
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/carousel/Indicator.svelte:12]: Yes, `-1` is reserved for temporary appearance of the last slide on the first-to-last transition. And other way around for `slides.length + 1`.

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/carousel/Indicator.svelte:12]: Gotcha. Maye just add this to the comment?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/carousel/Indicator.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/carousel/Indicator.svelte:12]: Sure, added.

**Pull request merged**